pom.xml

```diff
@@ -247,25 +251,8 @@ under the License.
             </excludes>
           </configuration>
         </plugin>
-        <!-- remove with parent pom upgrade -->
-        <plugin>
-          <groupId>org.apache.maven.plugins</groupId>
-          <artifactId>maven-project-info-reports-plugin</artifactId>
-          <version>3.3.0</version>
```

Comment:
Double check that the used MSITE is compatible with MPIR

Reply:
It is. We take versions from parent:
- maven-site-plugin:3.12.0
- maven-project-info-reports-plugin:3.3.0

all reports after `mvn site -P reporting` looks ok